eng/Workarounds.targets

```diff
@@ -7,7 +7,7 @@
   <ItemGroup>
     <FrameworkReference Remove="Microsoft.AspNetCore.App" />
     <!-- Required because the Razor SDK will generate attributes -->
-    <Reference Include="Microsoft.AspNetCore.Mvc" Condition="'$(UsingMicrosoftNETSdkWeb)' == 'true' AND '$(TargetFrameworkIdentifier)' == '.NETCoreApp'" />
+    <Reference Include="Microsoft.AspNetCore.Mvc" Condition="'$(UsingMicrosoftNETSdkWeb)' == 'true' AND '$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND $(GenerateRazorAssemblyInfo) == ''" />
```

Comment:
May need to quote the property reference i.e. `'$(GenerateRazorAssemblyInfo)' == ''`